cli/pcluster/cfnconfig.py

```diff
@@ -151,19 +130,14 @@ def __init_config(self):
 
         if not os.path.isfile(config_file):
             if default_config:
-                print("Default config %s not found" % config_file)
-                print(
-                    "You can copy a template from here: %s%sexamples%sconfig"
-                    % (
-                        os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe()))),
-                        os.path.sep,
+                self.__fail(
+                    "Default config {0} not found.\nYou can copy a template from here: {1}{2}examples{2}config".format(
```

Comment:
is there a missing param in the format call?

Reply:
good catch, I pasted the `config_file `parameter in the wrong place, fixed.